llama_index/llms/vertex.py

```diff
@@ -212,19 +226,17 @@ def stream_chat(
             prompt=question,
             chat=True,
             stream=True,
+            is_gemini=self._is_gemini,
             params=chat_params,
             max_retries=self.max_retries,
-            **params
+            **params,
         )
 
         def gen() -> ChatResponseGen:
             content = ""
             role = MessageRole.ASSISTANT
             for r in response:
-                if "text" in r.__dict__:
```

Comment:
do we need to remove this

Reply:
yes we do, "text" is always part of the response. __dict__ shouldn't be used if not strictly needed. In this case, the response parsing wasn't working properly